router/postsRoutes.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const router = express.Router();
+const { getPosts, setPosts } = require('../controllers/postsController');
+
+
+
+
+router.get('/', getPosts);
+router.post('/', setPosts)
```

Comment:
what does setPosts mean? Always strive to use meaningful names. for example `createPosts` simply tells the user that we want to create a new post.